docs/api-markdown-documenter/render-api-documentation.js

```diff
@@ -21,10 +21,23 @@ const { buildNavBar } = require("./build-api-nav");
 const { renderAlertNode, renderBlockQuoteNode, renderTableNode } = require("./custom-renderers");
 const { createHugoFrontMatter } = require("./front-matter");
 
-const apiReportsDirectoryPath = path.resolve(__dirname, "..", "_api-extractor-temp", "_build");
-const apiDocsDirectoryPath = path.resolve(__dirname, "..", "content", "docs", "apis");
+async function renderApiDocumentation(apiVersionNum) {
```

Comment:
Would you mind adding some documentation here explaining how `apiVersionNum` is used? I just want to make sure the contract is clear for future readers :)

Reply:
Make sense! Adding the comments here!